<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagsDialog_q</class>
 <widget class="QDialog" name="TagsDialog_q">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tags</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="allTagsLabel">
     <property name="text">
      <string>All tags</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="currTagsLabel">
     <property name="text">
      <string>Current tags</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="allTagsListView">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="buttonWidget" native="true">
     <layout class="QVBoxLayout" name="buttonLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="topSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add &gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>&lt;&lt; Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="bottomSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QListView" name="currTagsListView">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="newTagLayout">
     <item>
      <widget class="QLabel" name="newTagLabel">
       <property name="text">
        <string>New Tag:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newTagLineEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newTagButton">
       <property name="text">
        <string>Create</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TagsDialog_q</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TagsDialog_q</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
